src/lib/comms/sol-oic-cbor.c

```diff
@@ -63,7 +63,7 @@ initialize_cbor_payload(struct sol_oic_map_writer *encoder)
 
     cbor_encoder_init(&encoder->encoder, encoder->payload,
         buf->capacity - offset, 0);
-    old_ptr = encoder->encoder.ptr;
+    old_ptr = encoder->encoder.data.ptr;
```

Comment:
You're right, sorry. This change has been applied on master, it is not a part of v0.3.2. Will you agree if I update the reference and the commit message to reflect that this change is needed to be compatible with the master branch of tinycbor? If you prefer, You can also close this pull request and wait for the next tinycbor release.